modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -27,6 +27,8 @@ management cluster:: An {product-title} cluster where the HyperShift Operator is
 
 management cluster infrastructure:: Network, compute, and storage resources of the management cluster.
 
+node pool:: A resource that contains the worker nodes. The control plane contains node pools. The worker nodes run applications and workloads.
```

Comment:
Can this instance of "worker" be replaced with "compute"? Generally, we want to use "compute" wherever we can (similar to "control plane" instead of the legacy term "master").

```suggestion
node pool:: A resource that contains the compute nodes. The control plane contains node pools. The compute nodes run applications and workloads.
```

Reply:
good one. checking with SMEs

Reply:
Yes there is no distinction with hosted control planes. Updated to `compute`

Reply:
awesome thanks!